shared_model/backend/protobuf/commands/impl/proto_command.cpp

```diff
@@ -86,7 +83,12 @@ namespace shared_model {
     }
 
     Command *Command::clone() const {
```

Comment:
Please add TODO to rework hierarchy, so that clone() can be removed.
Also dependency on logger in the model is not good, so we should remove clone ASAP